docs/docs/hilbert.md

```diff
@@ -113,17 +113,17 @@ This information can be extracted with the {meth}`~netket.hilbert.DiscreteHilber
 
 It should be now evident why NetKet distinguishes locally discrete/countable spaces from arbitrary (e.g: continuous) spaces: if we can index the local basis, we can perform many optimisations and write efficient kernels to compute matrix elements of operators, but also Monte-Carlo samplers will propose transitions in a very different way than in continuous spaces.
 
-You can also obtain the total size of the hilbert space by invoking {attr}`~netket.hilbert.DiscreteHilbert.n_states`, which in general is equivalent to calling `np.prod(hi.shape)`.
+You can also obtain the total size of the hilbert space by invoking {attr}`~DiscreteHilbert.n_states`, which in general is equivalent to calling `np.prod(hi.shape)`.
 
 ```python
 >>> hi.n_states
 96
 ```
 
-Do bear in mind that this attribute only works if the hilbert space is indexable ({attr}`~netket.hilbert.DiscreteHilbert.is_indexable`), which is True when it has a size smaller than $ 2^{64} $. 
+Do bear in mind that this attribute only works if the hilbert space is indexable ({attr}`~DiscreteHilbert.is_indexable`), which is True when it has a size smaller than $ 2^{64} $. 
```

Comment:
```suggestion
Bear in mind that this attribute only works if the Hilbert space is indexable ({attr}`~DiscreteHilbert.is_indexable`), which is true if it has a dimension smaller than $2^{64}$. 
```
`Hilbert.size` is the number of sites in NetKet, so `dimension` is probably the better term here, right?